src/sentry/models/organizationmemberinvite.py

```diff
@@ -77,6 +77,9 @@ class OrganizationMemberInvite(DefaultFieldsModel):
     __relocation_scope__ = RelocationScope.Organization
 
     organization = FlexibleForeignKey("sentry.Organization", related_name="invite_set")
+    # SCIM provisioning requires that the OrganizationMember object exist. Until the user
+    # accepts their invite, the OrganizationMember is a placeholder and will not be surfaced via API.
+    organization_member = FlexibleForeignKey("sentry.OrganizationMember")
```

Comment:
Adding this as not null will make our checker complain. Since there are no rows currently it's safe, so you can set `checked=False` in the migration and it should go through.